packages/tupaia-web/src/api/queries/useValidatedQuery.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+import { useNavigate } from 'react-router';
+import { UseQueryResult } from 'react-query';
+
+export const useValidatedQuery = (queryResult: UseQueryResult) => {
```

Comment:
I'm curious why you added this? I'm not sure if we need it for landing pages? They should return a 200 response code with a null response if there is no landing page. And I don't think we want to redirect to a 404 page if there is an error.

Reply:
Yeah I wasn't sure - it seemed useful, but I can remove it